content/onboarding.rst

```diff
@@ -129,10 +127,10 @@ The initial stage is over when all the Onboarding tickets from the milestone ded
 
 We are far from single-sign-on and while working on this project you will have many different accounts.
 
-For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations (Chevah and ProAtria) and our core team. Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
+For GitHub you can create a new account or use your current account and ask to be part of our GitHub organisations and our core team. Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.
```

Comment:
does this use semantic newlines ? :)

For the `Ensure that in git-config, your user.email is the Pro:Atria email when working in the Chevah project repositories.`  this should be moved into wiki,

For open source chevah.com contributions we should allow any email

Reply:
Maybe this can be a future onboarding task.